airflow/api_fastapi/execution_api/datamodels/taskinstance.py

```diff
@@ -165,6 +165,7 @@ class TaskInstance(BaseModel):
     try_number: int
     map_index: int = -1
     hostname: str | None = None
+    start_date: UtcDateTime
```

Comment:
You can get that from, the TaskInstance won't have that field and can't be mandatory.

https://github.com/apache/airflow/blob/ee785a89ba27a59246cdfcc0d83129b691a39f3e/airflow/api_fastapi/execution_api/datamodels/taskinstance.py#L36-L51

Reply:
Ok - moved it into context instead.